### PR TITLE
Add Jack Lawrence to stakeholders.

### PR DESCRIPTION
**Pull request by jackhl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ The initial stakeholders consists of:
 * Tom Doron (@tomerd, Apple)
 * Daniel Dunbar (@ddunbar, Apple)
 * Dan Appel (@danappelxx, Zewo)
+* Jack Lawrence (@\_jackhl, Apple)
```
Comment (BasThomas) [inline README.md]: I think the `\_` is a mistake?

Comment (jackhl) [reply README.md]: Oops, yes.

Comment (jackhl) [reply README.md]: Fixed.

**Comment by seabaylea** [comment]
For the Apple staffers we might have to start breaking down which area(s) you work on in the future. For now, welcome aboard Jack.

**Pull request merged**